content/en/docs/ibm/create-cluster-vpc.md

```diff
@@ -250,7 +250,7 @@ time setup. Future `vpc-gen2` clusters can reuse the same VPC/subnet(with attach
 
    Finally, attach the public gateway to the subnet:
    
-   ```shell
+   ```
    ibmcloud is subnet-update $SUBNET_ID --public-gateway-id $GATEWAY_ID
```

Comment:
Ideally, this should be consistent with the rest of the guides, so:

```
    ```shell
    ibmcloud is subnet-update $SUBNET_ID --public-gateway-id $GATEWAY_ID
    ````

    Example output:

    ```
    Updating subnet 0737-27299d09-1d95-4a9d-a491-a6949axxxxxx under account IBM as user new@user-email.com...
    ...
    ...
    ```
```

(Without the indentation)

Reply:
Hi @8bitmp3, thank you for the careful examination. 

Indentation here has a purpose to nest items according the bullet points. Without it, it will not render properly.